### PR TITLE
Clean up login

### PR DESCRIPTION
**Pull request by naomilwx**
- Remove placeholder elements in homepage
- Redirect to login page if there is no active session
- Add user icon and logout button to header

### CODE REVIEW

File: web/router.ex [hunk 1]
```diff
@@ -7,7 +7,6 @@ defmodule Booky.Router do
     plug :fetch_flash
     plug :protect_from_forgery
     plug :put_secure_browser_headers
-    plug :assign_current_user
```
Comment (fonglh) [inline web/router.ex:10]: I think this might be necessary for other controllers/views to get access to the logged in user.

Comment (naomilwx) [reply web/router.ex:10]: That's done int the `AuthenticateUser` plug

File: web/controllers/page_controller.ex [hunk 2]
```diff
@@ -1,5 +1,6 @@
 defmodule Booky.PageController do
   use Booky.Web, :controller
+  plug Booky.Plug.AuthenticateUser
```
Comment (fonglh) [inline web/controllers/page_controller.ex]: Can this go in the router instead? Or whoever adds controllers in future might forget to put this in.

File: web/plug/authenticate_user.ex [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+  def init(default), do: default
+
+  def call(conn, _default) do
+    current_user = get_session(conn, :current_user)
+    if current_user do
+      assign(conn, :current_user, current_user)
+    else
+      conn
+        |> put_flash(:error, 'You need to login in to view Booky')
```
Comment (fonglh) [inline web/plug/authenticate_user.ex]: You need to login ~~in~~ to view Booky.

Comment (naomilwx) [reply web/plug/authenticate_user.ex]: oops

File: web/controllers/page_controller.ex (hunk 2 above)
Comment (naomilwx) [reply web/controllers/page_controller.ex]: hmm. it doesn't apply to the oauth routes though... I'll see if I can chain pipelines

**Pull request merged**